provider/grpc/provider.go

```diff
@@ -0,0 +1,158 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/go-logr/logr"
+	"github.com/konveyor/analyzer-lsp/dependency/dependency"
+	"github.com/konveyor/analyzer-lsp/provider/lib"
+	pb "github.com/konveyor/analyzer-lsp/provider/lib/grpc"
+	"go.lsp.dev/uri"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+type grpcProvider struct {
+	Client pb.ProviderServiceClient
+	log    logr.Logger
+	ctx    context.Context
+	conn   *grpc.ClientConn
+}
+
+func NewGRPCClient() *grpcProvider {
```

Comment:
Should the connection string not be an input to this?

Reply:
It will, but for first pass, didn't handle breaking up the config

Reply:
captured in #142 Resolving